spec/jobs/start_training_mail_job_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe StartTrainingMailJob do
+  let(:template) { :start_training }
+
+  let(:included) do
+    create_list :user, 3, :registered, confirmed_at: 4.weeks.ago
+  end
+
+  let(:excluded) do
+    [
+      create(:user, :registered, confirmed_at: 2.months.ago),
+      create(:user, :registered, confirmed_at: 4.weeks.ago, module_time_to_completion: { alpha: 0 }),
+    ]
+  end
+
+  # 1 user receives the :start_training email template
```

Comment:
The annotation I thought would be useful was more to do with the specs that had extra activity in `before` blocks that was needed to prepare the state. 
I feel the shared context explains who gets emailed what template clearly, but when that is dependent upon extra stuff it might be harder to workout further down the line.

Reply:
[ac451d1](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/747/commits/ac451d1b124e2a16c4a4f27c7fdd23a335eb1e9f)